### DISCUSSION TITLE
current distance snap not working

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#27381] [Q&A] [answered]
![image](https://github.com/ppy/osu/assets/78441708/00563a35-7f30-404d-9888-b7d8f3f74847)

On development build, latest commits as of right now you cannot take current distance snap between 2 notes.

**Comment by Gabixel** [comment]
I don't know much about this feature personally, but looks like the same problem was reported a few days ago.
I'm starting to think that there really is something wrong then, if others start reporting it.
- https://github.com/ppy/osu/discussions/27342

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/27386
